lib/graphql/query/arguments.rb

```diff
@@ -102,6 +102,18 @@ class << self
 
       NO_ARGS = NoArguments.new({}, context: nil, defaults_used: Set.new)
 
+      # Convert this instance into valid Ruby keyword arguments
+      # @return [{Symbol=>Object}]
+      def to_kwargs
+        ruby_kwargs = {}
+
+        keys.each do |key|
+          ruby_kwargs[Schema::Member::BuildType.underscore(key).to_sym] = self[key]
+        end
+
+        ruby_kwargs
+      end
+
```

Comment:
I know some people are philosophically opposed to `tap`, but it's really great at building up hashes and then returning them automatically. May I suggest:

```ruby
def to_kwargs
  {}.tap do |kwargs|
    keys.each { |key| kwargs[Schema::Member::BuildType.underscore(key).to_sym] = self[key] }
  end
end
```

Reply:
actually in this case perhaps:

```ruby
keys.each_with_object({}) do |key, kwargs|
  kwargs[Schema::Member::BuildType.underscore(key).to_sym] = self[key]
end
```

Reply:
Thanks, I'm satisfied with it as is.